documentation/site/pages/provers/povw-claiming-rewards.mdx

```diff
@@ -0,0 +1,274 @@
+
+```bash
+export POVW_LOG_ID="0x0000...0000"
+export POVW_PRIVATE_KEY="..."
+```
+
+To stake from a separate wallet, provers may additionally specify:
+
+```bash
+export ZKC_WALLET_ADDRESS="0x0000...0000"
+export PRIVATE_KEY="..."
+```
+
+Provers can now follow the [staking instructions](/provers/povw-claiming-rewards#staking-zkc) to stake $ZKC from the `ZKC_WALLET_ADDRESS`. Once staked, provers must delegate rewards i.e. associate the `POVW_LOG_ID` with `ZKC_WALLET_ADDRESS`. This is done with the following command:
+
+```bash
+boundless zkc delegate-rewards ${POVW_LOG_ID}
+```
+
+This command uses the `PRIVATE_KEY` environment variable to send a transaction onchain, therefore funds are required on `ZKC_WALLET_ADDRESS`.
+
+Once the rewards are delegated, the prover can continue with the normal reward claiming workflow and they can [start proving](/provers/povw-claiming-rewards#start-proving).
+
+#### Separating the Staking Wallet from the Value Recipient
+
+Additionally, the prover can specify a separate Ethereum address to receive the $ZKC reward. By default this is the log ID, but you can override it with `--value-recipient <address>` when running `send‑update`. 
+
+#### Separating Keys when runnning a Boundless Prover
+
+Since many Boundless provers run clusters on hosted cloud providers, and because the private key must be in plaintext when starting the broker, the security of this wallet depends on your trust in the cloud provider. Therefore, for provers running large proving/ZK-mining clusters with large expected $ZKC reward amounts, it is not recommended to use your Boundless proving address as the value recipient. If the key is compromised, this can lead to a loss of rewards and require new wallet setup to resume proving. 
+
+At the minimum, we recommend most provers separate their value recipient address, such that:
+
+* Boundless Prover Address + Log ID Address can be the same  
+* Value Recipient is a separate address
+
+This setup allows the proving cluster to prove orders on the Boundless market and generate and post work log updates onchain to be eligible for rewards. It also means that after running `claim-reward`, the reward $ZKC is transferred to a separate address. Claiming rewards is a permissionless process and does not require access to the private key for the Log ID. 
+
+### Keeping the State File Safe
+
+`prove-update` uses a state file to track compressed work receipts. Loss of this state file **will result in the loss of all work that is not submitted**for the respective Log ID, meaning rewards for any work done since the last work log update (via `send-update`) **will be lost**. In this scenario, a new work log ID must be specified for Bento, and Bento should be restarted to allow further work to be recorded properly against the new Log ID. *To avoid this scenario, it is recommended to store the state file in a durable location.*
```

Comment:
I think we should link to this in the section where they first create the state file. Very important